### PR TITLE
19 create members page

### LINKED ISSUE #33: 19 - Create members page
[MEMBERS LIST PAGE](https://www.figma.com/file/MpfNES0KuBUkkd0Qk4EyOc/HYF-Connect-Wireframe?node-id=17%3A2)

### PR DESCRIPTION
**Pull request by Moamin-AbuEwaida**
members page has member ticket component, the component data now is static and should be changing dynamically after fetching data from the backend

### CODE REVIEW

**Review by Hajir-pro — DISMISSED** [review]

File: client/pages/members/memebers-ticket-section.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export const membersTicketSection = {
```
Comment (dirk-jacobs) [inline client/pages/members/memebers-ticket-section.js]: This is not really a Section but a component.
I would rename this to MemberComponent and the file to member-component.js

File: client/pages/members/members.js [hunk 2]
```diff
@@ -1,21 +1,32 @@
...
-        <main-section></main-section>
-        <footer-component></footer-component>
-      </div>
-    
-      `,
+<div class="page">
+<header-component></header-component>
+<main-section></main-section>
+<banner-section></banner-section>
+<members-ticket-section></members-ticket-section>
```
Comment (dirk-jacobs) [inline client/pages/members/members.js:19]: I would introduce a MemberSection which 'loops' over the different MemberComponent
As an example, check BenefitsSection in the homepage

**Review by dirk-jacobs — CHANGES_REQUESTED** [review]
There us still some work to do.
But even as a first version, I would like to see some changes.
Please check the benefits or testimonials in the homepage

**Comment by Moamin-AbuEwaida** [comment]
i edited the MembersComponents / MembersSection according to the  BenefitsSection in the homepage way
I'll start with backend connection so  MemberComponent loops through the users to call back the data from DB @dirk-jacobs

**Comment by Moamin-AbuEwaida** [comment]
I managed to fetch the user's data from the database as well as the users skill -added the functions to business logic, controller and routes user.js- also succeeded to present the needed data into the frontend part 
for not receiving errors in the console i commented the lines for the skills selection part as still work needed to be :thinking:

**Pull request merged**